data/helpers.d/backend

```diff
@@ -51,3 +51,41 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated nginx config
+#
+# usage: ynh_add_nginx_config
+ynh_add_nginx_config () {
+	finalnginxconf="/etc/nginx/conf.d/$domain.d/$app.conf"
+	ynh_compare_checksum_config "$finalnginxconf" 1
+	sudo cp ../conf/nginx.conf "$finalnginxconf"
```

Comment:
We can possibly add the source file as an optional argument.
But, I think the better way is to use one config file instead of 2. Even is you need to configure differently if your app is installed on root or on a subpath.

Like I did with searx.
https://github.com/YunoHost-Apps/searx_ynh/blob/a12c4b6e925023d30e04bf044f5580077a7a62e1/conf/nginx.conf
https://github.com/YunoHost-Apps/searx_ynh/blob/a12c4b6e925023d30e04bf044f5580077a7a62e1/scripts/install#L59-L66